src/book/mod.rs

```diff
@@ -246,6 +246,11 @@ impl MDBook {
 
     /// Run `rustdoc` tests on the book, linking against the provided libraries.
     pub fn test(&mut self, library_paths: Vec<&str>) -> Result<()> {
+        self.test_chapter(library_paths, None)
+    }
+
+    /// Run `rustdoc` tests on a specific chapter of the book, linking against the provided libraries.
```

Comment:
This is a little unclear to me on what happens if `chapter` is None. Can it include a little more explanation?

Reply:
Sure thing, I added clarifying comments `// test_chapter with chapter:None will run all tests.` and on the `test_chapter` function itself the doc comment: `/// If `chapter` is `None`, all tests will be run.`